Proguard Setup

@eveliotc . 

The debug version of my app works fine. I created a release build for my app, and on selecting a picture to be cropped the app crashes stating the following. 

    java.lang.RuntimeException: Failure delivering result ResultInfo{who=null, request=10001, result=-1,     data=Intent { dat=content://media/external/images/media/9279 }} to activity    {com.prism.prismapp/com.prism.prismapp.CropImageActivity_}: java.lang.IllegalStateException:    You must provide a BitmapLoader.
    at android.app.ActivityThread.deliverResults(ActivityThread.java:3367)
    at android.app.ActivityThread.handleSendResult(ActivityThread.java:3410)
    at android.app.ActivityThread.access$1300(ActivityThread.java:141)
    at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1260)
    at android.os.Handler.dispatchMessage(Handler.java:102)
    at android.os.Looper.loop(Looper.java:136)
    at android.app.ActivityThread.main(ActivityThread.java:5113)
    at java.lang.reflect.Method.invokeNative(Native Method)
    at java.lang.reflect.Method.invoke(Method.java:515)
    at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:793)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:609)
    at dalvik.system.NativeStart.main(Native Method)
    Caused by: java.lang.IllegalStateException: You must provide a BitmapLoader.
    at com.lyft.android.scissors.CropViewExtensions.resolveBitmapLoader(Unknown Source)
    at com.lyft.android.scissors.CropViewExtensions$LoadRequest.load(Unknown Source)
    at com.lyft.android.scissors.CropView$Extensions.load(Unknown Source)
    at com.prism.prismapp.g.onActivityResult(Unknown Source)
    at android.app.Activity.dispatchActivityResult(Activity.java:5440)
    at android.app.ActivityThread.deliverResults(ActivityThread.java:3363)

Can we have an updated read me with the proguard setup?

--- a/scissors/proguard-rules.txt
+++ b/scissors/proguard-rules.txt
@@ -1,5 +1,7 @@
 # Picasso
 -dontwarn com.squareup.picasso.**
+-keepnames class com.squareup.picasso.Picasso
 
 # Glide
 -dontwarn com.bumptech.glide.**
+-keepnames class com.bumptech.glide.Glide

